app/editor/src/features/admin/notifications/constants/defaultRazorTemplate.ts

```diff
@@ -3,9 +3,10 @@ export const defaultRazorTemplate = {
 @using System.Linq
 @{
   var isAV = @Content.ContentType == TNO.Entities.ContentType.AudioVideo;
+  var useSummary = @Content.ContentType == TNO.Entities.ContentType.Image || isAV;
```

Comment:
Include template change that will be part of a database migration in another PR.  We can't deploy a DB migration to PROD because there is one waiting in the queue already.